sdcm/cluster.py

```diff
@@ -806,12 +806,28 @@ def wait_jmx_down(self, verbose=True):
         wait.wait_for(func=lambda: not self.jmx_up(), step=60,
                       text=text)
 
+    def _report_housekeeping_uuid(self, verbose=True):
+        """
+        report uuid of test db nodes to ScyllaDB
+        """
+        uuid_path = '/etc/scylla.d/housekeeping.uuid'
+        report_status_path = '/etc/scylla.d/housekeeping.uuid.reported'
+        cmd = 'curl "https://i6a5h9l1kl.execute-api.us-east-1.amazonaws.com/prod/check_version?uu=%s&mark=scylla"'
+
+        if self.file_exists(uuid_path) and not self.file_exists(
```

Comment:
The check has to be done in the remote system, therefore using `self.remoter.run('test -e %s' % report_status_path)`